src/common/types.ts

```diff
@@ -139,3 +139,11 @@ export interface ICommandLookup {
 export interface IPluginLookup {
   [pluginName: string]: IPlugin;
 }
+
+export const Roles: {[role: string]: number} = {
+  "Suspended": -10,
+  "RegularUser": 0,
+  "Teaching Assistant": 20,
+  "Moderator": 30,
+  "Admin": 40,
+}
```

Comment:
saa

Reply:
roles is an array like ["dank", "Admin"] and when I have to index over the enum I can't index with random strings with TS

Reply:
nm joey said it might lemme check